halo2_proofs/src/plonk/evaluation.rs

```diff
@@ -442,14 +447,19 @@ impl<C: CurveAffine> Evaluator<C> {
                     }
                 });
             }
+            stop_measure(start);
 
             // Lookups
+            let start = start_measure("lookups", false);
             for (n, lookup) in lookups.iter().enumerate() {
                 // Polynomials required for this lookup.
                 // Calculated here so these only have to be kept in memory for the short time
                 // they are actually needed.
-                let product_coset = pk.vk.domain.coeff_to_extended(lookup.product_poly.clone());
-                let permuted_input_coset = pk
+                let product_coset = pk
+                    .vk
+                    .domain
+                    .coeff_to_extended(lookup.product_poly.clone());
+                let permuted_input_coset  = pk
```

Comment:
Unnecessary diff it looks like.

Reply:
Reverted.

Reply:
I think this one is not yet resolved.

Reply:
Not sure what happened here.  Should be reverted now.